CHANGELOG.asciidoc

```diff
@@ -8,6 +8,7 @@
 // Template, add newest changes here
 
 === Beats version HEAD
+https://github.com/elastic/beats/compare/v5.0.0-rc1...master[Check the HEAD diff]
```

Comment:
Or better yet, reset it and then add a line in the Added / Affecting all Beats section that contains something like:

`- Added decode_json_fields processor for decoding fields containing JSON strings. {pull}2605[2605]`